src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -668,6 +668,8 @@
 	public static final RegistryObject<BlockItem> STEEL_BEAM = registerBlockItem(MSBlocks.STEEL_BEAM, MSItemGroup.LANDS);
 	public static final RegistryObject<BlockItem> MYCELIUM_COBBLESTONE = registerBlockItem(MSBlocks.MYCELIUM_COBBLESTONE, MSItemGroup.LANDS);
 	public static final RegistryObject<BlockItem> MYCELIUM_STONE = registerBlockItem(MSBlocks.MYCELIUM_STONE, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> MYCELIUM_STONE_CRUXITE_ORE = registerBlockItem(MSBlocks.MYCELIUM_STONE_CRUXITE_ORE, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> MYCELIUM_STONE_URANIUM_ORE = registerBlockItem(MSBlocks.MYCELIUM_STONE_URANIUM_ORE, MSItemGroup.LANDS);
```

Comment:
Move this code to a line next to the other uranium and cruxite ores(respectively)